pkg/kudoctl/util/repo/testdata/flink-index.yaml.golden

```diff
@@ -2,7 +2,16 @@ apiVersion: v1
 entries:
   flink:
   - appVersion: 0.7.0
+    description: fancy description is here
+    digest: 0787a078e64c73064287751b833d63ca3d1d284b4f494ebf670443683d5b96dd
+    home: kudo.dev
+    maintainers:
+    - name: Fabian Baier <fabian@mesosphere.io>
```

Comment:
I think this should be 
name: XXX
email: XXX

Reply:
this would require parsing of the 0.5.0 operator format which is not done in this case... it will a parsable format in the near future.